src/combobox/example/index.ts

```diff
@@ -142,7 +142,9 @@ export class App extends WidgetBase<WidgetProperties> {
 	}
 
 	render(): DNode {
-		return v('div', [
+		return v('div', {
+			styles: { maxWidth: '256px' }
```

Comment:
i assume this is to restrict the width of the inputs, should they not have their own max width?